lib/shopping_cart.rb

```diff
@@ -0,0 +1,40 @@
+class ShoppingCart
+
+  attr_reader :name, :capacity, :products
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @products = []
+  end
+
+  def add_product(product)
+    @products << product
+  end
+
+  def details
+    store_details = {}
+    store_details[:name] = name
+    store_details[:capacity] = capacity.delete("items").to_i
+    store_details
+  end
+
+  def total_number_of_products
+    @products.sum do |product|
+     product.quantity
+   end
+  end
+
+  def is_full?
+    if capacity.delete("items").to_i < total_number_of_products
```

Comment:
you just need this line without the `if` it will automatically evalute to true or false.. no need for the if/else statement.